### DISCUSSION TITLE
Github on premises

### DISCUSSION THREAD

**Discussion by Raxso1** [community/community#75431] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi

I am investing gibhub for a client of mine, they want to use it on premises in an infrastructure that is isolated from the internet for code repos and CI\CD. my questions are.

1. Can it be installed on a local network?
2. Can it be integrated with Windows Active Directory?
3. What OS does it run on Windows or Linux or either?
4. Does it need a backend database?
5. Can it support multi-tenants on Vmware? and have different repos for different departments for developers.

**Comment by b8zeek** [comment, marked as answer]
Greetings. 👋🏼 

Yes, of course! There's GitHub Enterprise which offers everything that you need.

- Yes, it can be installed on a local network.
- It can it be integrated with Windows Active Directory, it supports integration with LDAP, SAML and CAS, which includes Windows Active Directory.
- You can run it on any system that support virtualization.
- GitHub Enterprise comes with its own database.
- No sure about this one... It is designed to be a single tenant. You sure can have different repositories for different departments.

Find more about the service on the link below:

https://github.com/enterprise

I would also advise reaching out to the sales, they will provide all the necessary information for you.

https://github.com/enterprise/contact?ref_cta=Contact+Sales&ref_loc=hero&ref_page=%2Fenterprise&scid=&utm_campaign=Enterprise&utm_content=Enterprise&utm_medium=referral&utm_source=github

Take care. 🍺 

🍀
